selfdrive/controls/lib/events.py

```diff
@@ -950,6 +950,11 @@ def joystick_alert(CP: car.CarParams, CS: car.CarState, sm: messaging.SubMaster,
     ET.NO_ENTRY: NoEntryAlert("Vehicle Sensors Calibrating"),
   },
 
+  EventName.actuatorsApiUnavailable: {
+    ET.PERMANENT: NormalPermanentAlert("Missing Actuators API", "Your car may not be supported"),
+    ET.NO_ENTRY: NoEntryAlert("Missing Actuators API", "Your car may not be supported"),
```

Comment:
```suggestion
    ET.PERMANENT: NormalPermanentAlert("Check Compatibility", "Vehicle lacks required features"),
    ET.NO_ENTRY: NoEntryAlert("Check Compatibility"),
```

how about something like this?